### PR TITLE
Class 13

### PR DESCRIPTION
**Pull request by rpcvjet**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1 @@
+export GITHUB_TOKEN=cd07c2a05169db85474a774bfd3eefa153d83c90
```
Comment (patci-zz) [inline .env:1]: ☝️   oops!

File: .gitignore [hunk 2]
```diff
@@ -1,3 +1,6 @@
 node_modules
 
 githubtoken.js
+
+env.sh
+.env
```
Comment (patci-zz) [inline .gitignore:6]: Good! You may need to remove your .env file locally before adding your new token

File: SCRIPTS/Models/repo.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+'use strict';
+(function (module) {
+  var reposObj = {};
+
+  reposObj.requestRepos = function(callback) {
+    // NOTE: refactor this request into an $.get call
+    $.when(
+      $.get('/github/users/rpcvjet/repositories'),
+      $.get('/github/users/rpcvjet/followers')
+    ).done(callback);
```
Comment (patci-zz) [inline SCRIPTS/Models/repo.js:10]: Very nice!  The above waits for both requests to finish 😎

File: .env (hunk 1 above)
Comment (rpcvjet) [reply .env:1]: Is the "export" incorrect? Or does the githubingore key have to be the same as the GITHUB_TOKEN?

Comment (patci-zz) [reply .env:1]: it's correct, just that I can see it on GitHub, which means they probably revoked it.

**Pull request merged**